### PR TITLE
build: v0.13.3 alpha

### PR DESCRIPTION
**Pull request by ellemouton**
Still waiting for local run of `make protos` to complete so pushing the PR so long for review. 

Bumps tapd -> https://github.com/lightninglabs/taproot-assets/tree/v0-4-1-lnd-18
Bumps loop -> v0.28.6-beta

### CODE REVIEW

**Comment by ellemouton** [comment]
oops ... fixing the break now

**Comment by guggero** [comment]
You need this: https://github.com/lightninglabs/lightning-terminal/blob/68ed0bd3cad737837dfe4a4c95cf25768e1fe765/subservers/taproot-assets.go#L198-L209

And you can pass `false` as the new `litdIntegrated` argument of `ConfigureSubServer`.

File: README.md [hunk 1]
```diff
@@ -78,6 +78,7 @@ in remote mode (meaning that `lnd-mode=remote` is set). It shows the
 
 | LiT               | LND          |
 |-------------------|--------------|
+| **v0.13.3-alpha** | v0.17.0-beta |
```
Comment (ViktorTigerstrom) [inline README.md:81]: Just noting that this should be changed after checking with @guggero offline :)

```suggestion
| **v0.13.3-alpha** | v0.17.1-beta |
```

**Review by ViktorTigerstrom — COMMENTED** [review]
Initial review looks good! Will test this once the protos have been updated and compilation errors has been addressed.

We should also cut the release notes with this PR :)!

**Review by guggero — APPROVED** [review]
LGTM 🚀

**Review by ViktorTigerstrom — COMMENTED** [review]
tACK LGTM :rocket::fire:!! Thanks a lot :pray:

**Review by ViktorTigerstrom — APPROVED** [review]

File: docs/release-notes/release-notes-0.13.3.md [hunk 2]
```diff
@@ -4,20 +4,26 @@
...
 ### Faraday
 
 ### Taproot Assets
 
+* Updated the integrated [Taproot Assets version to
+  v0.4.1-alpha](https://github.com/lightninglabs/lightning-terminal/pull/804).
+
 # Autopilot
 
 # Contributors (Alphabetical Order)
```
[the remark this answers is not in the document]
Comment (ellemouton) [reply docs/release-notes/release-notes-0.13.3.md:29]: i dont think this PR counts as a contribution imo 🙃

[the remark this answers is not in the document]
Comment (ViktorTigerstrom) [reply docs/release-notes/release-notes-0.13.3.md:29]: Ah ok 😄

**Pull request merged**